observability-lib/grafana/variables.go

```diff
@@ -104,7 +109,8 @@ func NewQueryVariable(options *QueryVariableOptions) *dashboard.QueryVariableBui
 			Text:     dashboard.StringOrArrayOfString{ArrayOfString: []string{options.CurrentText}},
 			Value:    dashboard.StringOrArrayOfString{ArrayOfString: []string{options.CurrentValue}},
 		}).
-		Sort(dashboard.VariableSortAlphabeticalAsc).
+		Sort(options.Sort).
+		Refresh(*options.Refresh).
```

Comment:
Note: The default option when adding a query variable on the UI is disabled. Unless we want our internal default to be alphabetical ascending.